src/categorical_algebra/FinCats.jl

```diff
@@ -65,6 +65,7 @@ Because object generators usually coincide with objects, the default method for
 function ob_generator end
 
 ob(C::FinCat, x) = ob_generator(C, x)
+hom(C::FinCat,x::Nothing) = x
```

Comment:
Not clear why this line was added and it's not being called in the tests, so just delete it?